library/src/main/java/com/novoda/downloadmanager/lib/FileDownloadInfo.java

```diff
@@ -347,6 +351,19 @@ public boolean isResumable() {
         return alwaysResume || (eTag != null && isNoIntegrity());
     }
 
+    public boolean shouldAllowTarUpdate(String mimeType) {
```

Comment:
this is a bit more than `shouldAllow..` which would be just the boolean, `allowTarUpdates`.

`boolean tarCanBeUpdated(String mimeType)` to encompass both whether it's allowed && possible?